client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js

```diff
@@ -0,0 +1,354 @@
+	}
+	const hideCancelDialogHandler = () => {
+		props.setVisibilityCancelConnectionDialog( siteId, carrier, false );
+	}
+	const updateShowUPSInvoiceFields = () => {
+		props.toggleShowUPSInvoiceFields( siteId, carrier );
+	}
+
+	const displayErrors = () => {
+		if ( fieldErrors.license_agreement ) {
+			return <div className="carrier-accounts__settings-error">
+				{ fieldErrors.license_agreement }
+				<button className="carrier-accounts__settings-cancel-dialog-close-button" onClick={ hideCancelDialogHandler } ><Gridicon icon="cross"/></button>
+			</div>;
+		}
+	}
+	const upsInvoiceFields = () => {
+		return <div className="carrier-accounts__settings-ups-invoice">
+			<div className="carrier-accounts__settings-two-columns">
+				<TextField
+					id={ 'ups_invoice_number' }
+					title={ translate( 'UPS invoice number' ) }
+					value={ getValue( 'ups_invoice_number' ) }
+					updateValue={ updateValue( 'ups_invoice_number' ) }
+					error={ fieldErrors.ups_invoice_number }
+				/>
+				<TextField
+					id={ 'ups_invoice_date' }
+					title={ translate( 'UPS invoice date' ) }
+					value={ getValue( 'ups_invoice_date' ) }
+					updateValue={ updateValue( 'ups_invoice_date' ) }
+					error={ fieldErrors.ups_invoice_date }
+				/>
+			</div>
+			<div className="carrier-accounts__settings-two-columns">
+				<TextField
+					id={ 'ups_invoice_amount' }
+					title={ translate( 'UPS invoice amount' ) }
+					value={ getValue( 'ups_invoice_amount' ) }
+					updateValue={ updateValue( 'ups_invoice_amount' ) }
+					error={ fieldErrors.ups_invoice_amount }
+				/>
+				<TextField
+					id={ 'ups_invoice_currency' }
+					title={ translate( 'UPS invoice currency' ) }
+					value={ getValue( 'ups_invoice_currency' ) }
+					updateValue={ updateValue( 'ups_invoice_currency' ) }
+					error={ fieldErrors.ups_invoice_currency }
+				/>
+			</div>
+			<TextField
+				id={ 'ups_invoice_control_id' }
+				title={ translate( 'UPS invoice control id' ) }
+				value={ getValue( 'ups_invoice_control_id' ) }
+				updateValue={ updateValue( 'ups_invoice_control_id' ) }
+				error={ fieldErrors.ups_invoice_control_id }
+			/>
+		</div>;
+	}
+
+	const cancelDialogButton = () => {
+		return [ <Button compact primary scary onClick={ () => history.back() }>{ translate( 'Cancel' ) }</Button> ];
+	}
+
+	return (
+		<div className="carrier-accounts__settings-container">
+			{ displayErrors() }
+			<div className="carrier-accounts__settings">
+				<div className="carrier-accounts__settings-info">
+					<h4 className="carrier-accounts__settings-subheader">{ translate( 'Connect your UPS account' ) }</h4>
+					<p className="carrier-accounts__settings-subheader-description">{ translate( 'Set up your own UPS carrier account to compare rates and print labels from multiple carriers in WooCommerce Services. Learn more about adding {{a}}carrier accounts{{/a}}.', { components: { a: <a href="https://link.to.carrier.accounts.com/" />  } } ) }</p>
```

Comment:
Similar to the above, do we have an issue to update the `https://link.to.carrier.accounts.com` link?

Reply:
I don't think so... I'll create one if we don't have it.

Reply:
If we know the link, we can just add it to this PR.

Reply:
It's probably this one? https://www.easypost.com/ups-guide Not 100% sure, @c-shultz do you know?

Reply:
I think it was going to be a doc page. I'll double-check on the issue: https://github.com/Automattic/woocommerce-services/issues/2065

Reply:
@Ferdev 
Let's just remove that `Learn more...` sentence and the link, per [this comment](https://github.com/Automattic/woocommerce-services/issues/2065#issue-656506211).

We'll re-add it once we have a document created.